cranelift/assembler-x64/src/api.rs

```diff
@@ -24,17 +24,13 @@ pub trait CodeSink {
     /// required for assembling memory accesses.
     fn add_trap(&mut self, code: TrapCode);
 
-    /// Return the byte offset of the current location in the code buffer;
-    /// required for assembling RIP-relative memory accesses.
-    fn current_offset(&self) -> u32;
+    /// Inform the code buffer of a use of `target` is about to happen within
```

Comment:
grammar fix -- "inform ... of a use ... is about to happen" -> "inform ... that a use ... is about to happen"